### DISCUSSION TITLE
Add/edit labels

### DISCUSSION THREAD

**Discussion by KevinRaney** [community/community#11096] [Mobile]
It would be nice if we could add/edit labels within the mobile app. 

Preferably, if we can make new labels on the fly when adding/editing an issue.

**Comment by mxie** [comment]
Thanks for your feedback @KevinRaney! Label management isn't currently on our roadmap but we'd be happy to consider if there's a high demand for it from users. We'll keep you posted if we start looking into this!

**Reply by atapas** [reply]
It will be a very helpful one to have, especially keeping events like Hacktoberfest in mind. We can add label to the repository also can create it for a repo to use for issues and PR!
